style/style.css

```diff
@@ -1,214 +0,0 @@
-/* class = .class */
-/* id = #id */
-  :root {
```

Comment:
I'd still like to use the color variables, can those get worked into the styles.css file?